deployment/commons/sso/keycloak/entrypoint/run-keycloak

```diff
@@ -28,9 +30,9 @@ if [ ! -f /opt/jboss/keycloak/standalone/data/first-run ]; then
   #
   # Start and wait Keycloak
   echo "Starting Keycloak..."
-  /opt/jboss/tools/docker-entrypoint.sh -b localhost &
+  KEYCLOAK_ADMIN=$KEYCLOAK_USER KEYCLOAK_ADMIN_PASSWORD=$KEYCLOAK_PASSWORD /opt/keycloak/bin/kc.sh start-dev --http-port=$KC_PORT &
```

Comment:
Is start-dev the best option here? The main advantange of keycloak's development mode is having HTTP enabled and strict hostname resolution disabled - do we need those? Otherwise I'd prefer to enforce https, even if it is a local machine.